### ISSUE TITLE
Installation issues

### ISSUE THREAD

**Issue by sumitkm** [edwork/homeassistant-peloton-sensor#5]
Hi I am a HA noob, so apologies first up, this might be more of a basic HA question. I am running Hass IO on Raspberry Pi (image provided officially).
After I followed the instructions to install this sensor, I keep getting the error `No module named "pylotoncycle"` 
I don't seem to have python environment via the Terminal integration, so Python is unavailable. 

I downloaded the Python module using curl and copied the *.py files to the `custom_components/peloton` folder, no dice. Before I start changing the import statements, is there a better and obvious way to do this that  I am missing?

Many thanks for the work on this integration. Cheers.

**Comment by edwork** [comment]
Hello,  

First, thanks for helping test the integration!

I believe that this problem happens because I am importing the entire `pylotoncycle` module, rather than the submodules. Home Assistant checks explicitly for the modules being used and throws this error. I'm confident that forcing a restart of your instance will bypass the check. When integrations are loaded they download their required modules via the `manifest.json` file belonging to this integration. 

TL;DR - Try forcing your instance to restart as the warning is technically incorrect. 

Having said all of that, I'll define the modules differently and try to eliminate this bug. 

Thanks again for the report!
Ed

**Comment by sumitkm** [comment]
Okay cool, I did a `reboot now` from the terminal and now the sensor is working. That wasn't too hard :-)

I had tried re starting the server a few times from the dashboard, that didn't quite work.

Thanks again.
